pkg/apis/service/v1alpha1/types.go

```diff
@@ -43,6 +46,27 @@ type FalcoServiceConfig struct {
 	// Configuration for custom webhook
 	// +optional
 	CustomWebhook *Webhook `json:"webhook,omitempty"`
+
+	// -------------------------------------------------------------------
+	// added due to issue #215
+
+	Rules *Rules `json:"rules,omitempty"`
+
+	Destinations *[]Destination `json:"destinations,omitempty"`
+}
+
+type Destination struct {
+	Name              string  `json:"name,omitempty"`
+	ResourceSecretRef *string `json:"resourceSecretRef,omitempty"`
+}
+
+type Rules struct {
+	StandardRules *[]string     `json:"standard,omitempty"`
+	CustomRules   *[]CustomRule `json:"custom,omitempty"`
+}
+
+type CustomRule struct {
+	ResourceRef string `json:"resourceRef,omitempty"`
```

Comment:
This should be ResourceName (i.e., the name of a resource in `.spec.resources[]` in the `Shoot`)

Reply:
good point, we will change that